step_implementations/GoFunctionalTests/context/Agents.rb

```diff
@@ -0,0 +1,91 @@
+##########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+require 'bundler'
+require 'process_builder'
+require 'os'
+require 'fileutils'
+
+module GoFunctionalTests
+  module Context
+    class Agents
+
+      include GOCD
+
+      def initialize
+        @gauge_agent_dir = "target/gauge_agents"
+      end
+
+      def start_command
+        OS.windows? ? "start-agent.bat" : "./agent.sh"
+      end
+
+      def stop_command
+        OS.windows? ? "stop-agent.bat" : "./stop-agent.sh"
+      end
+
+      def start_an_agent_in dir
+        raise "Agent already running in the directory #{dir}" if File.exists?("#{dir}/.agent-bootstrapper.running")
+        FileUtils.rm_rf(dir) if Dir.exists?(dir)
+
+        FileUtils.mkdir_p "#{dir}"
+        FileUtils.cp_r "#{GoConstants::AGENT_DIR}#{GoConstants::GO_VERSION}/.", "#{dir}/"
+        Bundler.with_clean_env do
+          process = ProcessBuilder.build(start_command(), "") do |builder|
```

Comment:
For methods that do not require an argument, it's convention to not use parens. So this should be `start_command` instead of `start_command()`